recipe/meta.yaml

```diff
@@ -61,6 +62,7 @@ requirements:
     - libpng >=1.6.23,<1.7
     - openblas 0.2.19|0.2.19.*     # [not win]
     - blas 1.1 {{ blas_variant }}  # [not win]
+    - ffmpeg                       # [not win]
```

Comment:
No because Windows OpenCV has it's own crazy builds that it automatically downloads that are built with GPL extensions or something. I don't care enough to patch OpenCV to force it to use our FFMPEG and since we don't even build FFMPEG for Windows I don't see any reason to at the moment.